docs/distribution/README.md

```diff
@@ -0,0 +1,923 @@
+        }
+      ]
+    }
+    ```
+
+2. **manifest digest for the wabbit-networks signature** `sha256:222mbbf80b44ce6be8234e6ff90a1ac34acbeb826903b02cfa0da11c82cb222m`
+
+    ```json
+    {
+      "schemaVersion": 2,
+      "mediaType": "application/vnd.oci.image.manifest.v2+json",
+      "config": {
+        "mediaType": "application/vnd.cncf.notary.config.v2+jwt",
+        "digest": "sha256:222cb130c152895905abe66279dd9feaa68091ba55619f5b900f2ebed38b222c",
+        "size": 1906
+      },
+      "layers": []
+    }
+    ```
+
+    The `manifest.config` contains the following signature information:
+
+    ```json
+    {
+        "signed": {
+            "mediaType": "application/vnd.oci.image.manifest.v2+json",
+            "digest": "sha256:111ma2d22ae5ef400769fa51c84717264cd1520ac8d93dc071374c1be49a111m",
+            "size": 528,
+            "references": [
+                "registry.wabbit-networks.com/net-monitor:v1"
+            ],
+            "exp": 1627555319,
+            "nbf": 1596019319,
+            "iat": 1596019319
+        },
+        "signature": {
+            "typ": "x509",
+            "sig": "UFqN24K2fLj...",
+            "alg": "RS256",
+            "x5c": [
+                "MIIDszCCApugAwIBAgIUL1anEU/..."
+            ]
+        }
+    }
+    ```
+
+3. **manifest digest for the acme-rockets signature** `sha256:333mc0c33ebc4a74a0a554c86ac2b28ddf3454a5ad9cf90ea8cea9f9e75c333m`
+
+    ```json
+    {
+      "schemaVersion": 2,
+      "mediaType": "application/vnd.oci.image.manifest.v2+json",
+      "config": {
+        "mediaType": "application/vnd.cncf.notary.config.v2+jwt",
+        "digest": "sha256:333cc44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b785c333c",
+        "size": 1906
+      },
+      "layers": []
+    }
+    ```
+
+    The `manifest.config` contains the following signature information:
+
+    ```json
+    {
+        "signed": {
+            "mediaType": "application/vnd.oci.image.manifest.v2+json",
+            "digest": "sha256:111ma2d22ae5ef400769fa51c84717264cd1520ac8d93dc071374c1be49a111m",
+            "size": 528,
+            "references": [
+                "registry.acme-rockets.com/net-monitor:v1"
+            ],
+            "exp": 1627555319,
+            "nbf": 1596019319,
+            "iat": 1596019319
+        },
+        "signature": {
+            "typ": "x509",
+            "sig": "UFqN24K2fLj...",
+            "alg": "RS256",
+            "x5c": [
+                "MIIDszCCApugAwIBAgIUL1anEU/..."
+            ]
+        }
+    }
+    ```
+
+**Pros with this approach:**
+
+* OCI Artifacts already supports manifest based artifacts, through the `manifest.config.mediaType`
+
+**Cons with this approach:**
+
+* Manifests have no means to reference other artifacts.
+* An alternative is required to link a target artifact with it's signature. Either through parsing the signature `manifest.config` object, or [a separate API for linking objects](#linking-signatures-to-artifacts).
```

Comment:
If a annotation is added to the manifest of signature, it's not required to parse the `manifest.config` object.  But there's a possibility of inconsistency.  i.e. the annotation says it's a signature for "artifact A" but the "reference" in the signature object says it's a signature for "artifact B"

Reply:
We've generally avoided using annotations for anything other than interesting metadata you can get when looking at an object. 
But, to your point, we could use annotations, but similar to parsing the config object, it's still new logic registry operators would have to implement. The focus on Index allows registries to leverage the same linking infrastructure they use for multi-arch manifests, and would use for other artifact types like CNAB.